src/app/dataset-view/additional-components/metadata-component/components/set-transform/components/engine-section/engine-section.component.spec.ts

```diff
@@ -55,7 +55,7 @@ describe("EngineSectionComponent", () => {
     }));
 
     it("should check init engine and image when currentSetTransformEvent is not null", fakeAsync(() => {
-        component.currentSetTransformEvent = mockCurrentSetTransform;
+        //    component.currentSetTransformEvent = mockCurrentSetTransform;
```

Comment:
You need to rework this. I did not notice this the last time, but the low-level component should not be dependent on `SetTransform` specific type. Instead, it should be getting the information it needs via more universal @Input.

Reply:
Fixed